docs/user/tutorial.rst

```diff
@@ -8,7 +8,7 @@ Tutorial
 Writing a Driver
 ----------------
 
-First look at what parameters are and how to create them: `qcodes parameter <https://github.com/QCoDeS/Qcodes/blob/master/docs/examples/Parameters.ipynb>`__ .
```

Comment:
We really need to update the user guide and tutorial asap

Reply:
yep .... :(